src/pages/instant-observability.js

```diff
@@ -65,11 +67,25 @@ const filterByContentType = (type) => (quickstart) => {
 
 /**
  * Filters a quickstart based on a category.
- * @param {String} type The category type (e.g. 'featured').
+ * @param {String} category The category type (e.g. 'featured').
  * @returns {(Object) => Boolean} Callback function to be used by filter.
  */
-const filterByCategory = (type) => (quickstart) => {
-  return !type || (quickstart.keywords && quickstart.keywords.includes(type));
+const filterByCategory = (category) => {
+  const associatedKeywords = CATEGORIES.find((cat) => cat.value === category)
+    ?.associatedKeywords;
```

Comment:
Looks like we do! This is the default config for all gatsby projects https://github.com/gatsbyjs/gatsby/tree/master/packages/babel-preset-gatsby

Reply:
Oh that's good to know. Either way, I think there's value in ensuring that `associatedKeywords` conforms to an array type (by supplying a `[]` default - this lets us only need to check length) rather than optional chaining, which will return `undefined`.

Reply:
Good call on the defaulting, it makes the function much more simple